app/views/site/_study_resources_sidebar.html.erb

```diff
@@ -0,0 +1,52 @@
+<nav id="study-sidebar-content">
+  <%= link_to "<i class='fas fa-angle-double-right' id='resource-btn'></i>".html_safe, '#',
+              id: 'toggle-study-resources', class: 'btn btn-sm btn-default' %>
+  <% if @study.authors.corresponding.any? %>
+    <h3>Corresponding authors</h3>
+    <% @study.authors.corresponding.each do |author| %>
+      <%= link_to "#{author.first_name} #{author.last_name} <i class='fa fas fa-envelope'></i>".html_safe, '#',
+                  class: 'btn btn-sm btn-default corresponding-author',
+                  data: {
+                    contact: author.base_64_email, placement: 'top',
+                    'analytics-name' => 'email-corresponding-author'
+                  },
+                  title: "Send email to #{author.first_name} #{author.last_name}" %>
+    <% end %>
+  <% end %>
+  <% if @study.publications.any? %>
+    <h3>Related publications</h3>
+    <% @study.publications.each do |publication| %>
+      <h4><%= link_to publication.title, publication.url, target: :_blank, rel: 'noopener noreferrer',
```

Comment:
It seems [`noreferrer` has been secure for almost 5 years](https://github.com/broadinstitute/single_cell_portal_core/pull/1328#discussion_r790874788): 

```suggestion
                    target: :_blank, rel: 'noopener', 'data-analytics-name' => 'publication-link-pmc' %>
```